src/ontology/mondo-edit.obo

```diff
@@ -234076,19 +234076,34 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/9189/charcot-
 
 [Term]
 id: MONDO:0011891
-name: epilepsy, childhood absence, susceptibility to, 2
-def: "Any childhood absence epilepsy in which the cause of the disease is a mutation in the GABRG2 gene." [MONDO:patterns/disease_series_by_gene]
+name: febrile seizures, familial, 8
+def: "Mutations in the GABRG2 gene cause a spectrum of seizure disorders, ranging from early-onset isolated febrile seizures (FS) to childhood absence epilepsy (CAE) to generalized epilepsy with febrile seizures plus, type 3 (GEFS+3), which tends to represent a more severe phenotype." [OMIM:607681]
```

Comment:
This isn't a very OBOish text def. Read my blog post on definitions. Definitions should be genus-differentia, not discursive pieces of text